app/src/features/projects/components/ProjectFundingForm.tsx

```diff
@@ -155,73 +160,95 @@ const ProjectFundingForm: React.FC<IProjectFundingFormProps> = (props) => {
                     )?.[0]?.label;
 
                     return (
-                      <ListItem disableGutters className={classes.fundingListItem} key={index}>
-                        <Paper variant="outlined">
-                          <Toolbar>
+                      <Card key={index} variant="outlined" className={classes.fundingSourceItem}>
```

Comment:
The funding source does have anything that can make it truly be unique. I did update it but sure enough if someone was willing they could just fill out the form in the same way. The first nations is a little tricky as well as nothing on the form is required so if they just pick the same group twice it will throw the warning. The DB id could be used but we'd only have that value when users are editing existing projects

Reply:
Gotcha, ya fair. I think as long as its unlikely its probably fine. In theory, even if they pick the same funding source, it should be because they have different amounts and/or cover different date ranges.

Reply:
A fair compromise could be to form a key using a combination of the index, name, id(?) or other information:

```jsx
key={`${fundingSource.id}-${fundingSource.investment_action_category_name}-${index}`}
```

Reply:
spoke to Nick about this and went with a uuid to guarantee unique items